sgp4/tests.py

```diff
@@ -31,6 +31,53 @@
 
 class FunctionTests(TestCase):
 
+    def test_tle_export(self):
+        """Check `export_tle()` round-trip using all the TLEs in the test file.
+
+        This iterates through the satellites in "SGP4-VER.TLE", generates `Satrec` objects and exports the TLEs.
+        These exported TLEs are then compared to the original TLE, closing the loop (or the round-trip).
+
+        """
+        tlepath = os.path.join(thisdir, 'SGP4-VER.TLE')
+        with open(tlepath) as tlefile:
+            tlelines = iter(tlefile.readlines())
+
+        for line1 in tlelines:
+
+            if not line1.startswith('1'):
+                continue
+
+            line2 = next(tlelines)
+
+            # trim lines to normal TLE string size
+            line1 = line1[:69]
+            line2 = line2[:69]
+            satrec = Satrec.twoline2rv(line1, line2)
+
+            # Generate TLE from satrec
+            out_line1, out_line2 = export_tle(satrec)
+
+            # print("file :  " + line1)
+            # print("satrec: " + out_line1)
+            #
+            # print("file :  " + line2)
+            # print("satrec: " + out_line2)
+
+            # Skip these lines, known errors
+            # Resulting TLEs are equivalent (same Satrec object) but they are not the same
+            # 25954: BSTAR = 0 results in a negative exp, not positive
+            # 29141: BSTAR = 0.13519 results in a negative exp, not positive
+            # 33333: Checksum error as expected on both lines
+            # 33334: Checksum error as expected on line 1
+            # 33335: Checksum error as expected on line 1
+            expected_errs_line1 = [25954, 29141, 33333, 33334, 33335]
```

Comment:
It will take a little time to rebuild each of these lists every time through this loop. So:

1. Move the definitions up outside the loop, above the `for`. That way, the lists only get created once.
2. The `in` operation on a list needs to loop through the whole sequence. Try using sets here instead. Thanks!

Reply:
1. That was me being silly, fixed.
2. Changed the following to `set`:
```
        expected_errs_line1 = set([25954, 29141, 33333, 33334, 33335])
        expected_errs_line2 = set([33333, 33335])
```